client/app/app.vue

```diff
@@ -71,48 +72,45 @@
       },
 
       created () {
-          const query = this.$router.currentRoute.query
-
-          // Show Dialogs
-          if (query.dialogs) {
-              this.dialogs = query.dialogs.split(',').map((id) => {
-                  return Dialogs.find(item => item.id === id)
-              })
-          }
+          this.updateDialogs()
       },
 
       watch: {
-          dialogs (val, oldVal) {
```

Comment:
👍 

You are totally right; this is useless since `updateDialogs ` is handled in `create` method and `watch:$route`.